### PR TITLE
Update prod.tfvars

### PR DESCRIPTION
**Pull request by nitinprabhu**

### CODE REVIEW

File: infrastructure/prod.tfvars [hunk 1]
```diff
@@ -1,7 +1,13 @@
 vault_section = "prod"
 capacity = "2"
+reupload_delay = "10000"
+reupload_enabled = true
+scan_delay = "10000"
+scan_enabled = true
```
Comment (doncem) [inline infrastructure/prod.tfvars]: booleans have to be wrapped in double quotes

File: infrastructure/prod.tfvars [hunk 2]
```diff
@@ -1,7 +1,13 @@
...
+
+orchestrator_notifications_task_enabled = "true"
+
 api_gateway_test_valid_certificate_thumbprint = "3B0D23863EE7EF90290AD02B0ACFD7C37ED5B330"
 allowed_client_certificate_thumbprints = ["919F8A770C2A2F06933BA11551001ED0581B8848"]
 
 blob_processing_delay_in_minutes = "30"
-error_notifications_enabled = "false"
```
Comment (doncem) [inline infrastructure/prod.tfvars]: are credentials and url updated for/in production vault?

Comment (nitinprabhu) [reply infrastructure/prod.tfvars]: yes done that

File: infrastructure/prod.tfvars (hunk 1 above)
Comment (nitinprabhu) [reply infrastructure/prod.tfvars]: done

**Pull request merged**